charts/home-assistant/1.0.29/templates/deployment.yaml

```diff
@@ -30,7 +30,10 @@ spec:
         {{- include "common.labels.selectorLabels" . | nindent 8 }}
       annotations: {{ include "common.annotations" . | nindent 8 }}
     spec:
+    {{- if .Values.hostNetwork }}
```

Comment:
Let's please move this to after `hostNetwork:` as once it's set and it's unset again - i don't think it will turn it off by default and the older value would carry on unless explicit `hostNetwork: false` is specified

Reply:
Not super clean, but for now will address the issue.

We can discuss to include this in common.
```yaml
{{- if .Values.hostNetwork }}
      hostNetwork: true
      dnsPolicy: ClusterFirstWithHostNet
{{- else }}
      hostNetwork: false
{{- end }}
```

Reply:
dnsPolicy should also be configurable outside of hostNetwork.
Maybe check copy the compatible 5 lines from TC Common for now?